webpack/config.node.js

```diff
@@ -1,10 +1,16 @@
 const merge = require('webpack-merge');
+const nodeExternals = require('webpack-node-externals');
 const baseConfig = require('./config.base.js');
 
 module.exports = merge(baseConfig, {
     target: 'node',
     output: {
         filename: 'index.node.js',
         libraryTarget: 'commonjs2'
-    }
+    },
+    externals: [
+        nodeExternals({
+            whitelist: ['axios', 'validate.js']
+        })
+    ]
```

Comment:
` externals: [
        nodeExternals({
            whitelist: ['axios', 'validate.js']
        })
    ],`

should also be in those files as well:
```
webpack/config.browser.js
webpack/config.reactNative.js
```